src/utils/logger.utils.ts

```diff
@@ -33,3 +33,5 @@ export const setLogLevel: (logLevel: string | undefined) => void = (
       break;
   }
 };
+
+export const shortSha = (sha: string) => sha.slice(0, 7);
```

Comment:
Somewhat duplicative with the logic in our monorepo ([source](https://github.com/alwaysmeticulous/meticulous/blob/main/packages/webapp-backend/src/github/github-pull-request/github-pull-request.service.ts#L339)). Is it worth moving to the sdk or too small?

Reply:
I think too small to be worth it